src/test/resources/player_moves_from_pipe.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Player attempts to move from a pipe to one of its neighbouring components.
```

Comment:
Could also add a feature for moving on a slippery pipe, but great scenarios nonetheless.